app/views/main/index.html.erb

```diff
@@ -1,3 +1,6 @@
+<%# Recommended keywords tag length: up to 255 characters, 20 words. %>
+<% set_meta_tags keywords: %w[writing author nanowrimo novel character fiction fantasy universe creative dnd roleplay larp game design] %>
```

Comment:
@drusepth These are the keywords I came up with off the top of my head, but I'm sure there are better ones. Feel free to modify this as you wish.